### PR TITLE
Voter cross-chain `Voted`/`Abstained` event syncing issue

### LINKED ISSUE #511: `LiquidityPoolAggregator` not found for `RootPool`
Image

This might be a syncing problem between states. The address above is a `RootPool` on OP that links to a pool on Lisk (chainId = 1135)

I suspect that Lisk chain wasn't being processed at the same timestamp as OP so this error has risen.

This happened on `Voted` event handler. Investigate more deeply the flow.

### PR DESCRIPTION
**Pull request by FonsSicca**
closes #511 

(should close above or at least help)

Implements:
- Deferred handling of `Voter.Voted` / `Voter.Abstained` when the corresponding leaf pool does not exist yet (creates `PendingVote`).
- Storing `PendingRootPoolMapping` in `RootCLPoolFactory.RootPoolCreated` when no matching leaf LiquidityPoolAggregator exists.
- Flushing pending mapping and votes in `CLFactory.PoolCreated `(create `RootPool_LeafPool`, apply all `PendingVotes` to the leaf pool).
- New schema types `PendingVote` and `PendingRootPoolMapping`.
- Integration/E2E tests for the cross-chain flow

## Summary by CodeRabbit

* **New Features**
  - Deferred pending-vote and pending-mapping storage with automatic flush when mappings become available.

* **Improvements**
  - Distinct failure reasons and safer handling for missing/ambiguous pool mappings.
  - Enhanced contextual error logging and normalized error messages.
  - New stable ID/hash utilities for mapping and pending-vote keys.

* **Bug Fixes**
  - Ensure pending votes are applied in chronological order and not lost during sync.

* **Tests**
  - Extensive new and expanded tests covering deferred flows, flushing, and edge cases.

### CODE REVIEW

File: src/EventHandlers/Voter/VoterCommonLogic.ts [hunk 1]
```diff
@@ -166,3 +167,50 @@ export function computeVoterRelatedEntitiesDiff(
...
+ * @param rootPoolAddress - The root pool address
+ * @param tokenId - The token ID
+ * @param weight - The weight of the vote
+ * @param eventType - The type of event (VOTED or ABSTAINED)
+ * @param timestamp - The timestamp of the event
+ * @param blockNumber - The block number of the event
+ * @param transactionHash - The transaction hash of the event
+ * @returns void
+ */
+export function createPendingVoteForDeferredProcessing(
```
Comment (FonsSicca) [inline src/EventHandlers/Voter/VoterCommonLogic.ts]: function responsible for the creation of `PendingVote` entity
